generators/client/templates/angular/package.json.ejs

```diff
@@ -179,13 +178,12 @@
     "pretest": "<%= clientPackageManager %> run lint",
     "test": "ng test --coverage --log-heap-usage -w=2",
     "test:watch": "<%= clientPackageManager %> run test -- --watch",
-    "watch": "concurrently 'npm run webapp:hmr'<% if(!skipServer) { %> 'npm run backend:start'<% } %>",
+    "watch": "concurrently npm:start<% if(!skipServer) { %> npm:backend:start<% } %>",
```

Comment:
Should have used `<%= clientPackageManager %>` instead.

`concurrently <%= clientPackageManager %>:start` will work for yarn? 🤔